### PR TITLE
Submission form > default language selection

### LINKED ISSUE #812: New submission > Default language not being set
### Description
On Staging, when submitting a new submission, the language dropdown is defaulting to EN. However, on submit, the error `\"language\": Expected \"String!\", found null."` appears and the submission is not submitted.

If you update the dropdown to another language and then update it again to EN, the submission works as expected.

### PR DESCRIPTION
**Pull request by clari182**
- resolves #812 

### How to test
1. Open [deploy preview](https://fix-language-select--clara-aiid-staging.netlify.app/)
2. Submit a new report without updating the language
3. You should be able to submit the report without issue
4. On the review page (ask Clara for login instructions), open your submission and change the language
5. Update submission
6. Re-open the submission form
7. The language you selected in step 4 should be selected on the language dropdown
8. Create a new incident
9. Incident creation should work as expected

### CODE REVIEW

**Comment by cesarvarela** [comment]
I think it will be easier to set a default value here: https://github.com/responsible-ai-collaborative/aiid/blob/master/site/gatsby-site/src/components/forms/SubmitForm.js#L31

**Comment by cesarvarela** [comment]
Oh but I think you can delete the other stuff!

**Comment by clari182** [comment]
@cesarvarela I have to leave the option selection otherwise when editing the submission (on the review page), it will always pre-select English, even if you selected another language upon submission.

File: site/gatsby-site/src/components/submissions/SubmissionForm.js [hunk 1]
```diff
@@ -275,14 +274,13 @@ const SubmissionForm = () => {
...
-            value={values.language}
-            onChange={handleChange}
-          >
+          <Form.Select name="language" placeholder="Report Language" onChange={handleChange}>
             {supportedLanguages.map((l) => (
-              <option key={l.code} value={l.code}>
+              <option
+                key={l.code}
+                value={l.code}
+                selected={values?.language && values.language === l.code ? true : false}
```
Comment (cesarvarela) [inline site/gatsby-site/src/components/submissions/SubmissionForm.js:282]: This line shouldn't be necessary 🤔

Comment (cesarvarela) [reply site/gatsby-site/src/components/submissions/SubmissionForm.js:282]: (The `selected` one)

**Pull request merged**